src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleaner.py

```diff
@@ -317,9 +317,21 @@ def _dispatchWflow(self, wflow):
             #       {REPLICATING, STUCK, SUSPENDED, WAITING_APPROVAL}.)
             #       We still need some proper logging for them.
             msg = "Skipping workflow: %s - tape transfers are not yet completed." % wflow['RequestName']
-            msg += " Will retry again in the next cycle."
+            msg += "Workflow in 'announced' state, hence proceeding only with MSTransferor / input data removal. "  
+            msg += "Will retry the remaining in the next cycle."
             self.logger.info(msg)
             self._checkStatusAdvanceExpired(wflow, additionalInfo=msg)
+            for pline in self.mstrlines:
+                try: 
+                    pline.run(wflow)
+                except Exception as ex:
+                    msg = f"{pline.name}: General error from pipeline"
+                    msg += "when cleaning input MSTransferor rules."
```

Comment:
Sorry to report this, but can you please properly space these strings concatenation? Otherwise the error message in the logs won't look great.